test_util/aws.py

```diff
@@ -363,7 +363,6 @@ def get_vpc_host_ips(self):
     'coreos': SSH_INFO['coreos']
 }
 
-
```

Comment:
This newline should stay. `OS_AMIS` has no relation to the above `SSH_INFO` dicts.